scripts/build.sh

```diff
@@ -5,7 +5,7 @@ set -ex
 # Constructor should be >= 3.0.1 for aarch64.
 # See https://github.com/conda-forge/miniforge/pull/2#issuecomment-554394343
 echo "***** Install constructor *****"
-conda install -y constructor>=3.0.1
```

Comment:
Same here I would prefer to keep `3.0.1`. `tqdm` will go back to `noarch` at some point and `constructor>=3.0.1` will be required. I know the selector does not prevent to have a lower version installed but `3.0.1` is coming very soon.

Reply:
The old script was redirecting the output to a file called "=3.0.1" .... so....

Reply:
oh I didn't know that! So how do you think can we ask `>=3.0.1` with `"conda install -y constructor>=3.0.1"` maybe?

Reply:
I get that you want to use noarch packages, but I just don't think it is a necessary pinning. I would prefer removing it

Reply:
The thing is that when `tqdm` will be `noarch`, it will be picked up for building the installer. If at the same time a lower version than `3.0.1` of `constructor` is picked up then the installer won't build. Now I agree when `constructor=3.0.1` will be released, there is few chances that `conda` picks a lower version than `3.0.1` but you never know what a DAG will do and I prefer to be conservative.

Reply:
Feel free to specify it with "constructor>=3.0.1" but you will only be able to test 10% of the build today, and much of your hard work was in the other 90%

```
***** Install constructor *****
+ conda install -y 'constructor>=3.0.1'
Collecting package metadata (current_repodata.json): done
Solving environment: failed with initial frozen solve. Retrying with flexible solve.
Collecting package metadata (repodata.json): done
Solving environment: failed with initial frozen solve. Retrying with flexible solve.

PackagesNotFoundError: The following packages are not available from current channels:

  - constructor[version='>=3.0.1']

Current channels:

  - https://conda.anaconda.org/conda-forge/linux-aarch64
  - https://conda.anaconda.org/conda-forge/noarch
  - https://conda.anaconda.org/c4aarch64/linux-aarch64
  - https://conda.anaconda.org/c4aarch64/noarch
  - https://repo.anaconda.com/pkgs/main/linux-aarch64
  - https://repo.anaconda.com/pkgs/main/noarch
  - https://repo.anaconda.com/pkgs/r/linux-aarch64
  - https://repo.anaconda.com/pkgs/r/noarch

To search for alternate channels that may provide the conda package you're
looking for, navigate to

    https://anaconda.org

and use the search bar at the top of the page
```

Reply:
I don't think this PR will be merged soon anyway. So I am fine waiting for the new `constructor` version.